service/dap/server.go

```diff
@@ -1221,6 +1221,15 @@ func stoppedGoroutineID(state *api.DebuggerState) (id int) {
 // due to an error, so the server is ready to receive new requests.
 func (s *Server) doStepCommand(command string, threadId int, asyncSetupDone chan struct{}) {
 	defer s.asyncCommandDone(asyncSetupDone)
+	// All of the threads will be continued by this request, so we need to send
+	// a continued event so the UI can properly reflect the current state.
```

Comment:
It's good we can accomplish what we need this way, but seems like this was not the intended use based on the description in the spec. Do you think it would make more sense to have AllThreadsContinued added to Step/Next/etc responses?

Reply:
That would be great, since all of our commands that continue execution will resume all threads.

This would need to be added to a many responses, since only `ContinueRequest` includes the option and there are many flavors of commands that resume execution (ReverseContinue, StepOut, Next, StepInstruction, etc).

Reply:
https://github.com/microsoft/debug-adapter-protocol/issues/192